### PR TITLE
feat(api): add NewConfig func

### PR DESCRIPTION
**Pull request by bacongobbler**
fixes the issue where `deis routing:disable` sends an empty response object.

supercedes https://github.com/deis/controller-sdk-go/pull/73

see https://github.com/deis/workflow-cli/pull/170 for the client-side implementation.

### CODE REVIEW

**Comment by Joshua-Anderson** [comment]
I like this approach better, thanks for changing it!

I think the pointer solution has merit and we should discuss using it globally in the future. It's just confusing if we only did that with one field.

File: api/config.go [hunk 1]
```diff
@@ -45,7 +45,11 @@ type Config struct {
 	// It changes every time the configuration is changed and cannot be updated.
 	UUID string `json:"uuid,omitempty"`
 	// Routable determines if the application should be exposed by the router.
-	Routable bool `json:"routable,omitempty"`
+	Routable bool `json:"routable"`
+}
+
+func NewConfig() Config {
+	return Config{Routable: true}
```
Comment (kmala) [inline api/config.go]: if have a config object with Routable as `False`, now if i do `config:set` for some env variable wouldn't it send routable as `True` along with the env values?

Comment (bacongobbler) [inline api/config.go]: That is true. I guess that means we have to go the original route with using a bool pointer instead.

**Comment by bacongobbler** [comment]
closing since this approach won't work.

**Pull request closed without merging**